### PR TITLE
easy include everything

### PR DESCRIPTION
**Pull request by shicholas**
For people that just want everything (including the flex grid).

### CODE REVIEW

**Comment by shicholas** [comment]
if you think it's a good idea I can fix the tests.

**Comment by hellapixels** [comment]
Are you talking about the Travis CI tests?
@shicholas

**Comment by shicholas** [comment]
Yes

**Comment by hellapixels** [comment]
Yeah, if you are familiar with Travis have a go at it. We have been meaning to make some time for it here at the office, just busy with so many things. Was about to hack at it Today(which is why I was checking things out here) but if you would like to contribute that, it would be Awesome!
@shicholas

File: lib/generators/foundation/templates/foundation_and_overrides.scss [hunk 1]
```diff
@@ -7,7 +7,10 @@
 //
 // @import 'motion-ui/motion-ui';
 
-// We include everything by default.  To slim your CSS, remove components you don't use.
+// To include everything with the flex grid. Or choose from below
+@include foundation-everything(true);
```
Comment (ncoden) [inline lib/generators/foundation/templates/foundation_and_overrides.scss]: This will generate the CSS twice if untouched.

**Pull request closed without merging**